.travis.yml

```diff
@@ -1,9 +1,9 @@
-language: boa
```

Comment:
This file shouldnt be changed.  Grab the latest one from master.

Reply:
I think you cherry picked this change from master.
The heads of this file are the same on both branches.